### ISSUE TITLE
Small visual shadow glitch with the books

### ISSUE THREAD

**Issue by stephanie-walter** [smashingmagazine/redesign#287]
Meouw :) This is just a small little detail but designers get easily bothered by small details :D 

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**- What is the current behavior?**

Currently books presented into the [printed books section](https://www.smashingmagazine.com/printed-books/) (and inside articles) have 2 shadows, the book image shadow and the CSS one:

![screenshot 39](https://user-images.githubusercontent.com/741338/33178692-52f7cb50-d067-11e7-9559-cec60524f0b2.png)

The hover version is okay:
![screenshot 40](https://user-images.githubusercontent.com/741338/33178703-5dc9ece8-d067-11e7-9aa3-b1618bcbb95a.png)

**- What is the expected behavior?**
The non hover version of the book should not have the right shadow

**- Please mention your operating system version and the version of your browser**
mac siera chrome Version 62.0.3202.94 (Official Build) (64-bit)

**Comment by vitalyfriedman** [comment]
Correct! Will be fixed :-) Thanks for reporting it! @stephanie-walter
